.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: "Build"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  make-check:
+    runs-on: ubuntu-latest
+    container: fedora:latest
+    permissions:
+      security-events: write
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      id: dependencies
+      uses: ./.github/actions/install-dependencies
+
+    - name: Configure sssd
+      uses: ./.github/actions/configure
+
+    - name: make
+      shell: bash
+      working-directory: x86_64
+      run: |
+        source ../contrib/fedora/bashrc_sssd
+        make CFLAGS+="$SSS_WARNINGS -Werror"
+
+    - name: make check
+      shell: bash
+      working-directory: x86_64
+      run: |
+        source ../contrib/fedora/bashrc_sssd
+        make CFLAGS+="$SSS_WARNINGS -Werror" check
+
+    - name: make distcheck
+      shell: bash
+      working-directory: x86_64
+      run: |
+        source ../contrib/fedora/bashrc_sssd
+        AUX_DISTCHECK_CONFIGURE_FLAGS+=--with-smb-idmap-interface-version=5 make distcheck
+
+    - name: Get distro name
```

Comment:
This step is redundant since we only have one distro now, it can be removed.

Reply:
Removed, thanks.